js/views/PageNav.js

```diff
@@ -67,6 +66,10 @@ export default class extends View {
     window.history.forward();
   }
 
+  navReload() {
+    remote.getCurrentWindow().reload();
```

Comment:
Ok, I see why it's bombing. You need to change:

```
import remote from 'electron';
```

to

```
import { remote } from 'electron';
```

otherwise in the former, you're essentially importing the full 'electron' module, just assigning it to a variable named 'remote', whereas in the latter you're assigning the 'remote' property of the 'electron' module to a variable named 'remote'.